python/lsst/ts/tcpip/testone_client_read_loop_server.py

```diff
@@ -21,25 +21,61 @@
 
 import asyncio
 import logging
+from typing import Any
 
-from .constants import LOCAL_HOST
+from .base_client_or_server import ConnectCallbackType
 from .one_client_read_loop_server import OneClientReadLoopServer
 
 __all__ = ["TestOneClientReadLoopServer"]
 
 
 class TestOneClientReadLoopServer(OneClientReadLoopServer):
-    """An implementation of OneClientReadLoopServer for unit test
-    purposes.
+    """A simple implementation of OneClientReadLoopServer for unit tests.
+
+    Parameters
+    ----------
+    host : `str` or `None`
+        IP address for this server; typically `LOCALHOST_IPV4` for IPV4
```

Comment:
Would you like to mention the new `LOCALHOST` constant here?

Reply:
Good suggestion. Done for all server classes.